src/main/java/com/mgmtp/radio/dto/station/SongDTO.java

```diff
@@ -0,0 +1,25 @@
+package com.mgmtp.radio.dto.station;
+
+import com.mgmtp.radio.dto.user.UserDTO;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@NoArgsConstructor
+@Data
+public class SongDTO {
+	String id;
+	boolean isPlayed;
```

Comment:
isPlayed should be 'playing'. The part 'is' will make matter with spring data query